src/events/messageCreate.ts

```diff
@@ -11,8 +13,12 @@ export default TypedEvent({
     eventName: "messageCreate",
     run: async (client: Bot, message: Message) => {
         if (message.author.bot) return;
-        const foundPhrase = forbiddenPhrases.find((phrase) =>
-            message.content.toLowerCase().includes(phrase.toLowerCase())
+
+        const messageWords = weirdToNormalChars(message.content.toLowerCase()).split(" ");
+        const foundPhrase = forbiddenPhrases.some((phrase) =>
+            messageWords.join(" ").includes(phrase.toLowerCase())
+            || stringSimilarity(messageWords.join(" "), phrase) > 0.6
+            || messageWords.some((word) => stringSimilarity(word, phrase) > 0.6)
```

Comment:
How efficient is this, and is it really necessary? Last I checked we're only really filtering out invite links anyway (although I suppose for further expansion later, or others' forks, this could be useful)

Reply:
`string-similarity-js` uses the **Srensen–Dice coefficient**, and the author claims that "the algorithm is most effective at detecting rearranged words or misspellings, It tends to be less effective with very short strings, unless perhaps you switch to comparing individual characters in common instead of bigrams." which is convenient to me. Users do not always type words perfectly without small mistakes, so filtering misspelled `forbiddenPhrases` is necessary in my opinion.